src/controllers/verify-yc.ts

```diff
@@ -0,0 +1,98 @@
+import { Body, Controller, Ctx, Post } from 'amala'
+import { Context } from 'vm'
+import { badRequest } from '@hapi/boom'
+import { ethers, utils } from 'ethers'
+import { polygonProvider } from '@/helpers/providers'
+import AddressVerifyBody from '@/validators/AddressVerifyBody'
+import Attestation from '@/models/Attestation'
+import BalanceUniqueVerifyBody from '@/validators/BalanceUniqueVerifyBody'
+import EmailUniqueVerifyBody from '@/validators/EmailUniqueVerifyBody'
+import TwitterBody from '@/validators/TwitterBody'
+import Verification from '@/models/Verification'
+import fetchUserProfile from '@/helpers/twitter/fetchUserProfile'
+import getBalance from '@/helpers/getBalance'
+import sendEmail from '@/helpers/sendEmail'
+import signAttestationMessage from '@/helpers/signatures/signAttestationMessage'
+import zeroAddress from '@/models/zeroAddress'
+
+@Controller('/verify-yc')
+export default class VerifyYCController {
+  @Post('/email-unique')
+  async sendUniqueEmail(
+    @Body({ required: true }) { email }: EmailUniqueVerifyBody
+  ) {
+    const { message, signature } = await signAttestationMessage(
+      Attestation.YC,
+      Verification.email,
+      utils.hexlify(utils.toUtf8Bytes(email))
```

Comment:
We're doing `utils.hexlify(utils.toUtf8Bytes(...))` in many places, should we extract it into `helpers`?